.gitbook/contents/hc/3_write-smart-contract.md

```diff
@@ -0,0 +1,190 @@
+# Write the Smart Contract
+
+Now we can write the smart contract, which will call our previously created off-chain handler. Our contract has two purposes:
+
+1. Intentionally burn ETH gas (in order to test gas estimation logic).
+2. Count how many times the contract has been called based on certain inputs and errors.
+
+You can find the needed `HybridAccount` contract along with its dependencies in our [repository](https://github.com/bobanetwork/account-abstraction-hc/contracts/samples/HybridAccount.sol). Additionally, the contract is pulled in as a submodule when you checkout our [`rundler-hc` repository](https://github.com/bobanetwork/rundler-hc).
```

Comment:
The submodule path is rundler-hc/crates/types/contracts/lib/account-abstraction/...